biogrid/apps/api/src/app/services/Biogrid.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { simulateNewBiogrid } from './Biogrid';
+import { GRID_ITEM_NAMES } from '@biogrid/biogrid-simulator';
+
+describe('weather', () => {
+  test('that simulateNewBiogrid placed the proper amount of items on the grid', async () => {
+    const startDate = new Date();
+    const endDate = new Date();
+    startDate.setDate(startDate.getDate() - 2);
+    endDate.setDate(endDate.getDate() - 1);
```

Comment:
I mentioned this in another PR as well - why -1 and -2? 
can you do:
`const endDate = new Date(); const startDate = endDate.getDate() - 1;` instead?